modules/docs/src/main/tut/docs/refined-integration.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: docs
+title: Refined integration
+---
+
+### {{page.title}}
+
+Atto benefits from an integration with the excellent [refined](https://github.com/fthomas/refined)
+library.
+
+#### Installation
+
+The refined integration lives in the `atto-refined` module:
+
+```scala
+libraryDependencies += "org.tpolecat" %% "atto-refined" % "{{site.attoVersion}}"
+```
+
+#### Usage
+
+We'll need the usual imports as well as a specific `atto-refined` import:
+
+```tut:silent
+import atto._, Atto._, syntax.refined._
+import eu.timepit.refined.numeric._
+```
+
+We can refine any `Parser[T]` with a predicate `P` to obtain a `Parser[T Refined P]`, for example
+if we pick up our `int` parser and refine it to parse only positive integers:
+
+```tut
+val positiveInt = int.refined[Positive]
+```
+
+It will be able to parse positive integers:
+
+```tut
+positiveInt parseOnly "123"
+```
+
+And will fail on negative integers:
+
+```tut
+positiveInt parseOnly "-123"
```

Comment:
Maybe we can show that the error message will be the same as `refined` produces?

Reply:
Do you mean textually calling out that it's the error message as reported by refined? because the refined error message will definitely appear:

![img](https://screenshotscdn.firefoxusercontent.com/images/e13029c4-7aba-478d-8930-fb926f7945a0.png "abc")